config/configuration.md

```diff
@@ -252,6 +252,18 @@ The fetch interval in seconds between retrievals of signed API data.
 
 A list of signed API URLs to call along with URLs fetched from the chain.
 
+### `useSignedApiUrlsFromContract` _(optional)_
+
+A boolean flag that determines whether to use signed API URLs from the AirseekerRegistry contract. When set to `true`,
```

Comment:
```suggestion
A boolean flag that determines whether to use Signed API URLs from the AirseekerRegistry contract. When set to `true`,
```

Reply:
You're right. I went over whole repository here [97345b7](https://github.com/api3dao/airseeker/pull/452/commits/97345b75b6ff0014fd410ea61e9dd3e212a74414).